src/validators/sysNumTimestamp.ts

```diff
@@ -1,34 +1,35 @@
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { addHttpHeaders } from '../util/httpHeaders';
 
 // eslint-disable-next-line consistent-return
-export const validateGetErrors = (event: APIGatewayProxyEvent): APIGatewayProxyResult | undefined => {
+export const validateSysNumTimestampPathParams = (event: APIGatewayProxyEvent) => {
   if (!event.pathParameters?.systemNumber) {
```

Comment:
I've renamed this as it feels like there might be more than one path that needs to get systemNumber and Timestamp from the path parameters